socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -16,8 +16,8 @@ AnnotateMozCrashReason
 ashmem
 __ascii_stricmp
 __assert
-app_process@0x
-AppleIntelHD3000GraphicsGLDriver@
```

Comment:
We're not including the module offset, so we can drop all variations of `@0x` from the end of these frame signatures.

Note that we keep the two `@0x...` cases at the top of this file because we still have one case where it could be just an offset.